module/userProjects/helper.js

```diff
@@ -2917,16 +2941,14 @@ module.exports = class UserProjectsHelper {
 				}
 
 				//  fetch user data using userId of project and calling the profile API
-				let userProfileData = await userService.profileReadPrivate(userProject[0].userId)
+				let userProfileData = await userService.profile(userProject[0].userId)
 				if (
 					userProfileData.success &&
 					userProfileData.data &&
-					userProfileData.data.response &&
-					userProfileData.data.response.firstName &&
-					userProfileData.data.response.firstName !== ''
+					userProfileData.data.name &&
+					userProfileData.data.name !== ''
 				) {
-					userProject[0].userProfile.firstName = userProfileData.data.response.firstName
-					userProject[0].userProfile.lastName = userProfileData.data.response.lastName
+					userProject[0].userProfile.name = userProfileData.data.name
```

Comment:
@Prajwal17Tunerlabs please verify this works

Reply:
the user data fetch is verified.